src/font/shaper/run.zig

```diff
@@ -263,11 +271,7 @@ pub const RunIterator = struct {
             }
 
             // Add all the codepoints for our grapheme
-            try self.addCodepoint(
-                &hasher,
-                if (cell.codepoint() == 0) ' ' else cell.codepoint(),
-                @intCast(cluster),
-            );
+            try self.addCodepoint(&hasher, if (cell.codepoint() == 0) ' ' else cell.codepoint(), @intCast(cluster));
```

Comment:
Why change the formatting here?

Reply:
zig fmt, don't really like the change but didnt want to get held up in the check.

Reply:
Add a trailing comma and `zig fmt` will format it the way it was before.

Reply:
@jcollie thank you! I couldn't figure out why that magically changed on me.